### ISSUE TITLE
Decide on revenue model

### ISSUE THREAD

**Issue by erbridge** [to-the-end/to-the-end#98]
We should agree on how we split proceeds of the game, in the unlikely case that we make any money from it. My suggestion:

- 50% for everyone involved in the game jam, split evenly (so 8 ways)
- 50% for everyone involved in finishing the game off for first release, also split evenly
- Reevaluate changes to this after the initial release

Alternatively, we can agree to only ever release it for free. Either is acceptable to me.

It would be good for everyone to weigh in on this one, even if they're not interested in carrying on working on the game.

cc @endritbajo @mhle @daveallenbpm @duncjam @2easy4sam @Archeks @kanianto

**Comment by Archeks** [comment]
Is there any scheme with donation options? For instance, if we release it
as a free opensource, and there would be same program on a market to donate.

**Comment by erbridge** [comment]
itch.io supports Pay What You Want, yes (in fact it's default is to set a minimum price). But, I'm more talking about what we do with the income, than how we choose to make money from the thing.

**Comment by daveallenbpm** [comment]
To be honest, I'm not all that fussed about making money, but presuming that someone does throw a few pennies our way every once in a while (via itch donation or whatever platform we end up on) @erbridge, your suggestion sounds a fair model to me

**Comment by mhle** [comment]
@erbridge Happy to go with your suggestion for now.

**Comment by kanianto** [comment]
it looks a fair deal for me

**Comment by duncjam** [comment]
yeah sounds fair to me too. For my two cents, in terms of making money, I'm not particularly fussed about trying to earn from this project really - I'm happy just trying to make it what we want it to be, and then if we make money, awesome.

**Comment by erbridge** [comment]
Totally agree. Let's make something we believe in, and money is incidental.

**Comment by erbridge** [comment]
@endritbajo, @Archeks, @2easy4sam: I assume you're all fine with this?

**Comment by Archeks** [comment]
Sorry for late response. Yes im absolutely fine with that
